### DISCUSSION TITLE
Tabs on Show pages

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#580] [Feature requests]
Sometimes, when you have a model with a lot of associations you'll end up with a long Show page with a lot of screens. That can be bad for performance and experience altogether. 

Tabs could help with that.

**Comment by MrJoy** [comment]
I've found tabs useful in admin systems for more than just associations as well.  For example: Showing charts.  Showing a more complete view of the record than the default view, when there are fields that are only useful in uncommon workflows.

**Reply by adrianthedev** [reply]
I know what you mean and they are on the roadmap to be implemented soon. Thanks!

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/tabs.html#tabs-on-show-view
